packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -4211,104 +4211,192 @@ describe("database test", async () => {
         expect(writtenPayload.documentId).toEqual(mockDocument.id);
         expect(writtenPayload.eservice).toEqual(expectedEservice);
       });
-      it("should throw eServiceNotFound if the eservice doesn't exist", async () => {
-        expect(
-          catalogService.updateDocument(
-            mockEService.id,
-            mockDescriptor.id,
-            mockDocument.id,
-            { prettyName: "updated prettyName" },
-            {
-              authData: getMockAuthData(),
-              correlationId: "",
-              serviceName: "",
-              logger: genericLogger,
-            }
-          )
-        ).rejects.toThrowError(eServiceNotFound(mockEService.id));
-      });
-      it("should throw operationForbidden if the requester is not the producer", async () => {
+      it("should write on event-store for the update of a document for deprecated descriptor", async () => {
```

Comment:
if the body of these tests is always the same, I suggest [this](https://github.com/pagopa/interop-be-monorepo/blob/main/packages/purpose-process/test/archivePurposeVersion.test.ts#L209) approach